website/templates/project/contributors.mako

```diff
@@ -353,8 +356,10 @@
 
                 <!-- ko ifnot: contributor.canEdit() -->
                     <!-- ko if: canRemove -->
```

Comment:
The `contributors.mako` template uses [`canEdit`](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/templates/project/contributors.mako#L56), [`contributor.canEdit()`](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/templates/project/contributors.mako#L313) ([+](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/templates/project/contributors.mako#L328) [3](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/templates/project/contributors.mako#L344) [more](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/templates/project/contributors.mako#L357)), and [`$data.canEdit()`](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/templates/project/contributors.mako#L338). Which is which? Are they the same or different?